pkg/api/mlflow/dao/models/namespace.go

```diff
@@ -8,12 +8,12 @@ import (
 
 // Namespace represents model to work with `namespaces` table.
 type Namespace struct {
-	ID                  uint   `gorm:"primaryKey;autoIncrement"`
-	Code                string `gorm:"unique;index;not null"`
-	Description         string
-	CreatedAt           time.Time
-	UpdatedAt           time.Time
-	DeletedAt           gorm.DeletedAt `gorm:"index"`
-	DefaultExperimentID *int32         `gorm:"not null"`
-	Experiments         []Experiment   `gorm:"constraint:OnDelete:CASCADE"`
+	ID                  uint           `gorm:"primaryKey;autoIncrement" json:"ID"`
```

Comment:
why are we using custom json encoding for this instead of the defaults?

Reply:
it happened, because of `linter` wants to see them, so that's why I just added them. we serialise this object when send it via `pg_notify`.
<img width="1436" alt="Screenshot 2023-11-07 at 16 28 20" src="https://github.com/G-Research/fasttrackml/assets/7852635/dbd4c9f3-bebb-4412-aa2e-0ed0c3f411bd">